packages/frontend/src/stores/messagelist.ts

```diff
@@ -276,6 +276,27 @@ class MessageListStore extends Store<MessageListState> {
       }, 'messageChanged')
     },
     setMessageState: (messageId: number, messageState: number) => {
+      const messageLoaded = this.state.messageCache[messageId] != undefined
+      if (!messageLoaded) {
```

Comment:
```suggestion
      if (this.state.messageCache[messageId] === undefined) {
```
nitpick: here i would consider the condition easier to read without variable